src/main/java/uk/gov/hmcts/juror/support/sql/entity/jurorresponse/DigitalResponse.java

```diff
@@ -0,0 +1,118 @@
+package uk.gov.hmcts.juror.support.sql.entity.jurorresponse;
+
+import jakarta.persistence.Column;
+import jakarta.persistence.Entity;
+import jakarta.persistence.Table;
+import jakarta.persistence.Version;
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.EqualsAndHashCode;
+import lombok.Getter;
+import lombok.Setter;
+import uk.gov.hmcts.juror.support.generation.generators.value.DateFilter;
+import uk.gov.hmcts.juror.support.generation.generators.value.EmailGenerator;
+import uk.gov.hmcts.juror.support.generation.generators.value.LocalDateGenerator;
+import uk.gov.hmcts.juror.support.generation.generators.value.RegexGenerator;
+import uk.gov.hmcts.juror.support.sql.Constants;
+import uk.gov.hmcts.juror.support.sql.entity.AbstractJurorResponse;
+
+import java.time.LocalDate;
+import java.time.temporal.ChronoUnit;
+
+
+@Entity
+@Builder
+@AllArgsConstructor
+@Getter
+@Setter
+//@DigitalThirdPartyOtherReason
+@Table(name = "juror_response", schema = "juror_mod")
+@EqualsAndHashCode(callSuper = true)
+public class DigitalResponse extends AbstractJurorResponse {
+
+    @Column(name = "residency")
+    @Builder.Default
```

Comment:
Needs default using @FixedValueGenerator.
(Same for all fields with no generator)